tests/views/graph_manager_tests.py

```diff
@@ -470,3 +471,18 @@ def test_graph_import(self):
         self.assertEqual(imported_json[0], [])
         self.assertEqual(imported_json[1]["graphs_saved"], 1)
         self.assertEqual(imported_json[1]["name"], "Cardinality Test Model")
+
+    def test_prevent_card_save_of_published_graph(self):
+        """
+        test to make sure that a card cannot be saved if the graph is published
+
+        """
+
+        self.client.login(username="admin", password="admin")
+        user_id = self.client.session["_auth_user_id"]
+        logged_in_user = get_user_model().objects.get(pk=user_id)
```

Comment:
If we're logging in as admin anyway, seems less roundabout:
```suggestion
        logged_in_user = get_user_model().objects.get(username="admin")
```

Reply:
Oh I see it's that way in surrounding tests. No big deal then

Reply:
Actually, since I'm not using the client to perform any kind of action, I'll use your suggestion.